optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -1463,11 +1463,16 @@ def prepare_inputs_for_generation(
         }
 
     def _get_logits_processor(self, generation_config: GenerationConfig, *args, **kwargs):
-        forced_decoder_ids = generation_config.forced_decoder_ids
         # Whisper uses forced_decoder_ids for default task and language specification, while original _get_logits_processor does not allow it
         # see for details https://github.com/huggingface/transformers/issues/37172
-        if is_transformers_version(">=", "4.50.0"):
-            generation_config.forced_decoder_ids = None
-        logits_processor = super()._get_logits_processor(generation_config, *args, **kwargs)
-        generation_config.forced_decoder_ids = forced_decoder_ids
+        if not hasattr(generation_config, "forced_decoder_ids"):
```

Comment:
should also be enforced for transformers >= 4.53 no ? (even in the case where the generation config was saved with a forced_decoder_ids)

Reply:
I don't think we should modify the model's config in optimum (should be done in transformers if this is something expected by transformers)